src/Client.php

```diff
@@ -30,6 +31,7 @@ class Client
     use HandlesTasks;
     use HandlesKeys;
     use HandlesSystem;
+    use HandlesMultiSearch;
```

Comment:
Just interesting - what's the decision to use so many traits that are used only once here and that just hides the code?

Reply:
I'm following the adopted pattern, but in fact, I agree because the `Client` class would be humongous if all those methods were defined directly there.

Reply:
Hey @norkunas, 
@brunoocasali left on holliday so I'll be releasing this PR. I agree with you as well, nonetheless I suggest we open an issue on this subject for a possible refacto of that section and merge this one as it is. What do you think?

Reply:
hi @bidoubiwa, i don't mind :)